backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -71,18 +71,32 @@ private void updatePersonFacility(Person person, UUID facilityId) {
   }
 
   // Specifications filters for queries
+  private Specification<Person> setOrgFilter(Organization org) {
+    return (root, query, cb) -> cb.equal(root.get(SpecField.ORGANIZATION), org);
+  }
+
   private Specification<Person> inCurrentOrganizationFilter() {
     return (root, query, cb) ->
         cb.equal(root.get(SpecField.ORGANIZATION), _os.getCurrentOrganization());
   }
 
-  private Specification<Person> inAccessibleFacilitiesFilter(boolean includeArchived) {
-    Set<Facility> facilities = _os.getAccessibleFacilities();
-    if (includeArchived) {
-      facilities.addAll(_os.getArchivedFacilities());
+  private Specification<Person> inAccessibleFacilitiesFilter(
+      boolean includeArchived, Organization org) {
+    Set<Facility> facilities;
+    if (org != null) {
```

Comment:
⚠️ needed this check here since `getAccessibleFacilities()` and `getArchivedFacilities()` methods assume there is a current org